src/__tests__/hooks/easy.useCalendarView.spec.ts

```diff
@@ -4,21 +4,92 @@ import { useCalendarView } from '../../hooks/useCalendarView.ts';
 import { assertDate } from '../utils.ts';
 
 describe('초기 상태', () => {
-  it('view는 "month"이어야 한다', () => {});
+  it('view는 "month"이어야 한다', () => {
+    const { result } = renderHook(() => useCalendarView());
+    expect(result.current.view).toBe('month');
+  });
 
-  it('currentDate는 오늘 날짜인 "2024-10-01"이어야 한다', () => {});
+  it('currentDate는 오늘 날짜인 "2024-10-01"이어야 한다', () => {
+    const { result } = renderHook(() => useCalendarView());
+    const expectingDate = new Date('2024-10-01');
+    assertDate(expectingDate, result.current.currentDate);
+  });
```

Comment:
p5;
mocking 한 날짜의 위치도 경로를 주석으로 추가해주면 해석이 좀더 편할 것 같습니다 :) 

```suggestion
// @mock system date src/setupTests.ts#L19
  it('currentDate는 오늘 날짜인 "2024-10-01"이어야 한다', () => {
    const { result } = renderHook(() => useCalendarView());
    const expectingDate = new Date('2024-10-01');
    assertDate(expectingDate, result.current.currentDate);
  });
```